RECIPES.md

```diff
@@ -451,4 +451,30 @@ fun captureMethodExample() {
 }
 ```
 
+---
+
+## Providing a custom comparison method
+
+You can customize the algorithm used to compare the captured bitmap against the baseline by using the `setCompareMethod()` on `ScreenshotRule`.
+
+`setCompareMethod()` is provided with the baseline and current bitmaps and expects a boolean result. If you return true, the bitmaps are
```

Comment:
```suggestion
`setCompareMethod()` is provided with the baseline and current bitmaps and returns a boolean result. If you return true, the bitmaps are
```
The wording makes it sound a bit like method consumers are also providing the result to the method.